src/layouts/partials/WalletContent.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import { Paper } from 'material-ui'
+import { SendTokens, DepositTokens, TransactionsTable } from 'components'
+
+import { getAccountTransactions } from 'redux/wallet/actions'
+
+import styles from 'layouts/partials/styles'
+
+import './WalletContent.scss'
+
+export class WalletContent extends Component {
+
+  static propTypes = {
+    getTransactions: PropTypes.func,
+    tokens: PropTypes.object,
+    ready: PropTypes.bool,
+    isFetching: PropTypes.bool,
+    transactions: PropTypes.object,
+    endOfList: PropTypes.bool
+  }
+
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return !this.props.ready ? null : (
+      <div styleName="root">
+        <div styleName="content">
+          <div>
+          <div className="WalletContent__grid">
+            <div className="row">
+              <div className="col-sm-4 col-md-3 col-lg-3 col-xl-2" styleName="head-light">
+                <Paper style={styles.content.paper.style}>
+                  <SendTokens title="Send tokens" />
+                </Paper>
+              </div>
+              <div className="col-sm-6 col-md-3 col-lg-3 col-xl-4">
+                <div styleName="instructions">
+                  <h3>Title send tokens lorem</h3>
+                  <div styleName="description">
+                    <p>
+                      Once the printer ink runs dry it has to be replaced
+                      with another inkjet cartridge. There are many reputed
+                      companies like Canon, Epson, Dell, and Lexmark that
+                      provide the necessary cartridges to replace the empty
+                      cartridges.
```

Comment:
I have no assumptions about help text structure right now.
It may be a paragraph or a list with several points or a group of links, etc.
I can move it out of the markup in the resource strings "as is"